xgcm/grid.py

```diff
@@ -1807,15 +1829,17 @@ def cumint(self, da, axis, **kwargs):
         da_i : xarray.DataArray
             The cumulatively integrated data
         """
+        boundary = kwargs.get("boundary")
```

Comment:
This is inconsistent with the way you parse boundary and fill_value in `integrate`. They should all have the same functional form - either get them from kwargs or define them as explicit keyword arguments. Any thoughts which is preferrable @dcherian @andersy005 ?

Reply:
I always prefer explicit keyword arguments. Nice for autocomplete.

Reply:
`kwargs` in `cumint` is passed on to `cumsum` which can accept boundary and fill_value
`kwargs` in `average` and `integrate` are passed on to `xarray.mean` and `xarray.sum`
if I just include boundary and fill_value as part of `kwargs` for average and integrate I get the following error statements:
```python
TypeError: nansum() got an unexpected keyword argument 'boundary'
TypeError: mean() got an unexpected keyword argument 'boundary'
```
See lines 1813 and 1865